doc/infrastructures.rst

```diff
@@ -66,6 +66,8 @@ These filters are used to check whether a label value is matching a string. Stri
 
 * ``label_identifier != "wanted_value"`` will match if the label with the given name has not ``wanted_value`` as a value. Example : ``somename != "somevalue"``
 
+Please note that when used through Yorc CLI interface, the filter has to be between simple quotes, and the filter value has to be between simple quotes : ``yorc hp list -f 'somename="someval"'`` is a valid command, while ``yorc hp list -f somename="someval"`` and ``yorc hp list -f "somename='someval'"`` are not.
```

Comment:
Maybe here you can just set a reference to the CLI documentation and move this paragraph the CLI section

Reply:
I would think this is more logic to put it in there:
- The CLI interface doc refers to this already
- The quotes problem may eventually occur outside of the CLI in the future (lets say another interface), so it is probably better to pack it with filtering ?